api/scpca_portal/templates/readme/additional_terms/research_academic_only.md

```diff
@@ -1,3 +1,13 @@
 ### Additional Restrictions
 
-This dataset is designated as research or academic purposes only.
+{% if projects|length > 1 %}
+|Project ID|Data Usage Restrictions|
+|:---------|:--------------------|
+{% for project in projects %}
+|[{{ project.scpca_id }}](https://scpca.alexslemonade.org/projects/{{ project.scpca_id }})|{{ project.additional_restrictions }}|
+{% endfor %}
+{% else %}
+{% if additional_terms %}
+{{ additional_terms }}.
+{% endif %}
```

Comment:
This could be an elseif and then include the template directly.

Reply:
I've clean up the template context `additional_terms`from all create readme methods (at 9f4faf4b8f4826892d64ca23a2c8ce65e5d3a777), and since we'll no longer use this variable name `additional_terms` in their corresponding readme template files, I've also made the following updates:
- `include` the additional terms readme file in all readme files without if check, and because of this, we no longer need to use `Project::get_additional_terms` method to grab that template contents (at 8517f0b9a45f3cc03653bfadf32646e1609245a9)
- added the Additional Restrictions section's rendering condition to the additional terms readme file c2a033d43403c73b5b712724fb056be6fc8b8ef6.

I've also fixed the failing tests due to tag conflicts (e.g., `with` and `includes`) and now this PR is ready for another look, thank you David!